src/taco-remote-lib/ios/iosBuild.ts

```diff
@@ -73,6 +75,50 @@ class IOSBuilder extends Builder {
         return Q({});
     }
 
+    protected collectVersionInfo(): Q.Promise<any> {
+        var self: IOSBuilder = this;
+
+        // Get the version of cordova-ios and XCode currently installed
+        let execDeferred: Q.Deferred<string> = Q.defer<string>();
+        child_process.exec("xcodebuild -version & cordova platforms version ios", { cwd: this.currentBuild.appDir } , function (error, stdout, stderr) {
+            if (error) {
+                execDeferred.reject(new Error(resources.getString("XCode83GetIosPlatformVersionWarning")));
+            } else {
+                execDeferred.resolve(stdout.toString());
+            }
+        });
+
+        return execDeferred.promise.then(function parseVersionsFromOutput(execOutput: string) {
+            // Parse XCode version from output
+            const xcodeVersionRegex = /^Xcode ((\d+)(?:\.)(\d+))?/;
+            const xcodeMatch = execOutput.match(xcodeVersionRegex);
+            if (xcodeMatch[1]){
+                self.xcodeVersion = parseFloat(xcodeMatch[1]);
```

Comment:
Is treating the number as a float appropriate? Why not leave it as a string and explicitly compare major/minor components?